### PR TITLE
git-clean: add -fd option

### PR DESCRIPTION
**Pull request by JibranKalia**
----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/git-clean.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 `git clean`
 
+- Delete directories that are not tracked by git:
```
Comment (agnivade) [inline pages/common/git-clean.md]: Move this 2 examples down.

**Comment by JibranKalia** [comment]
@agnivade Not sure if this addresses your comments.

**Comment by agnivade** [comment]
Hmm .. I guess so.

Also, a minor note: in future, you do not need to squash commits and force push. We have that option while merging a PR, and we usually take care of it. Squashing commits make it hard to review what changed within each changeset.

**Comment by JibranKalia** [comment]
Thank You @agnivade. Will keep that in mind.

**Pull request merged**